Scripts/build.py

```diff
@@ -156,7 +156,7 @@ def xcodebuild(scheme, workspace, config, commands):
 
 
 def check_string_output(command):
-    return subprocess.check_output(command).decode().strip()
+    return subprocess.check_output(command).decode('utf-8').strip()
```

Comment:
Oh the bug fix, sorry, I misread your comment. I'd be interested in an explanation though 😉. I can't see how this can trigger an encoding problem, given than most OS X CLI tools don't care about $LOCALE and friends...